modules/ai/connect/ai.php

```diff
@@ -503,5 +504,46 @@ public function get_layout_prompt_enhanced( $prompt, $context = [] ) {
 		);
 	}
 
+	public function get_history_by_type( $type, $page, $limit, $context = [] ) {
+		$endpoint = Module::HISTORY_TYPE_ALL === $type
+			? 'history'
+			: add_query_arg( [
+				'page' => $page,
+				'limit' => $limit,
+			], 'history/' . $type );
```

Comment:
why not just "history/$type"

Reply:
No reason and phpcs seems also ok with that. 
Gonna update this line.